tests/unit/modules/test_junos.py

```diff
@@ -4,11 +4,13 @@
 '''
 # Import python libs
 from __future__ import absolute_import, print_function, unicode_literals
+import os
 
 # Import test libs
 from tests.support.mixins import LoaderModuleMockMixin, XMLEqualityMixin
-from tests.support.mock import patch, mock_open
+from tests.support.mock import patch, mock_open, PropertyMock, call, ANY
 from tests.support.unit import skipIf, TestCase
+import six
```

Comment:
Please import `salt.ext.six` instead

Reply:
I have updated the same in latest commit.